source/common/filter/config_discovery_impl.h

```diff
@@ -271,22 +275,27 @@ class FilterConfigProviderManagerImpl : public FilterConfigProviderManagerImplBa
   }
 
 protected:
+  virtual void
+  validateDefaultConfig(const ProtobufWkt::Any& proto_config, const std::string& filter_config_name,
+                        Server::Configuration::FactoryContext& factory_context,
+                        bool last_filter_in_filter_chain, const std::string& filter_chain_type,
+                        const absl::flat_hash_set<std::string> require_type_urls) const PURE;
```

Comment:
nit: send `require_type_urls` by reference.

Reply:
Thank you